src/components/modules/maverick/CommandsCard.vue

```diff
@@ -140,6 +141,11 @@ export default {
   },
   methods: {
     createQuerys() {
+      console.log(this.verifyQuery(maverickShellQuery, 'dev'))
+      console.log(this.verifyQuery(navSatFixMutate, 'dev'))
+      this.clearAllVerifiedQueries()
```

Comment:
Resets any validation that may have taken place. Useful if -api updates it schema.